service/mapper/transitions.go

```diff
@@ -211,6 +214,13 @@ func (t *Transitions) BootstrapState(s *State) error {
 		}
```

Comment:
above in case `<-ctx.Done():` you may want to add: `workerErrors <- ctx.Err()` to indicate early quit due to a context cancelation just to be a bit safer.